.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:6.11.1
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run:
+          name: Dependencies
+          command: yarn
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./yarn-installations
```

Comment:
I'm not sure if this path is the ideal name choice, let me know if there's a better one!

And thank you for all your help with this, the pointers have really helped me break down and understand how Circle's configuration works. (Hopefully this is more along the lines of what you're looking for!)

Reply:
Ah, this should be the path to the files you want to cache—since we don’t have a `yarn-installations` file or directory, this won’t do much :P

In our case, it should probably be `./node_modules`.

An easy way to sanity-check this to look at the “restoring cache” step output in Circle:

<img width="1167" alt="screen shot 2017-10-23 at 11 48 05 pm" src="https://user-images.githubusercontent.com/74178/31928360-ba95312c-b84c-11e7-8b42-554a6643863b.png">

You can see that the size of the cache being restored is only 32 bytes, so it’s basically empty. We definitely have a lot more dependencies that that!

**Important side note:** now that it has a bad cache, you’ll have to do something to change the cache key to clear it out. I suggest updating [the license field in `package.json`](https://github.com/edgi-govdata-archiving/web-monitoring-ui/blob/master/package.json#L46), since it’s totally wrong (we use GPL v3, which should be called `GPL-3.0` in the `package.json` file). I didn’t even realize that was wrong until I just looked for something worth changing 😱